### ISSUE TITLE
[Refactor] `Utskott` as DB table

### ISSUE THREAD

**Issue by afroborg** [esek/ekorre#255]
This should probably have been done from the beginning as this will be a lot of work...
BUT - I think it's a necessary thing to do.

We should move away from the hard coded `utskott` enum in graphql and store utskott in the DB as well as relations to their respective posts.

This would be an improvement for many reasons:
1. No need to redeploy just to change a minor issue with naming etc.
> I know this is not something we do often but it's still a pain to regenerate schemas etc. if an enum changes
2. We could add a `description` field to the utskott column, so that "utskottsordförande" can edit their own description on the site.
3. probably more reasons

My thinking is that it should have fields like:
- `id` - pk
- `name` - Ex. Informationsutskottet
- `abbreviation` - Ex. InfU
- `shortDescription` - A short description that summarizes the utskott
- `description` or `articleLink` - Either we store the entire description, or we link it to an article that describes the utskott in more detail.
- `color`? - Not sure about this, but could be a fun idea to let them decide their own utskott-color.

**Comment by ginger51011** [comment]
Agreed. But yes, a bit of a pain. Would also make it possible to change things about NollU when desireable

Will most certanly need a DataLoader, but that's probably easy to add
